README.md

```diff
@@ -1,82 +1,27 @@
 # OCI Artifacts
 
-## Artifact Guidance Documents
+# Archive Notice
 
-1. [Artifact Author Guidance](./artifact-authors.md)
+This repository has been archived and is no longer maintained.
+For current guidance on using OCI to package artifacts, please see the [image-spec artifacts guidance documentation](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md).
 
-## Supporting Documents
-
-- [Term Definitions](./definitions-terms.md)
-
-## Project Introduction and Scope
-
-Container registries, implementing the [distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of [distribution][distribution]. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features.
-
-Applications and services typically require additional artifacts to deploy and manage, including [helm](https://helm.sh) for deployment and [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
-
-Utilizing the [manifest][image-manifest] and [index][image-index] definitions, new artifacts, such as the [Singularity project][singularity], can be stored and served using the [distribution-spec][distribution-spec].
-
-This repository provides a reference for artifact authors and registry implementors for supporting new artifact types with the existing implementations of distribution.
-More particularly this repository has been tasked by the [OCI TOB](https://github.com/opencontainers/tob/blob/master/proposals/artifacts.md) to serve 3 primary goals:
-
-1. **artifact authors** - [guidance for authoring new artifact types.][artifact-authors] Including a clearing house for well known artifact types.
-1. **registry operators and vendors** - guidance for how operators and vendors can support new artifact types, including how they can opt-in or out of well known artifact types. Registry operators that already implement `media-type` filtering will not have to change. The artifact repo will provide context on how new `media-type`s can be used, and how `media-type`s can be associated with a type of artifact.
-1. **clearing house for well known artifacts** - artifact authors can submit their artifact definitions, providing registry operators a list by which they can easily support.
-
-By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS).
+## Mission Update
```

Comment:
the act of archiving, and directing users to the new reopos is a bit redundant to. See proposed changes in #70